src/components/actions/ActionBarConnected.tsx

```diff
@@ -25,7 +25,7 @@ const mapStateToProps = (state: IReactVaporState, ownProps: IActionBarOwnProps):
 
   return {
     withReduxState: true,
-    actions: actionBar && actionBar.actions ? _.filter(actionBar.actions, (action: IActionOptions) => _.result(action, 'enabled')) : [],
+    actions: actionBar && actionBar.actions ? _.filter(actionBar.actions, (action: IActionOptions) => action.enabled || action.hideDisabled === false) : [],
```

Comment:
Is the ` === false` really necessary? I don't think it's really used elsewhere in the repo.

Reply:
I had a problem in the UTs when I used `!action.hideDisabled` in the ActionBarConnected.spec.tsx like the defaultProps were not applied. This solved the problem.